lib/twitch-soundboard/views/index.pug

```diff
@@ -6,4 +6,20 @@ html(lang="en")
 		script(src='/js/soundboard.js')
 		link(href='/css/soundboard.css' rel='stylesheet' type='text/css')
 	body(bgcolor=transparent)
-		button#gift hpc gift!
+		<fieldset class="divider1">
```

Comment:
Can you convert Legend and Fieldset tags to PUG format instead of HTML?

e.g. 
legend.title1
fieldset.divider1